Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TextToolIntegrationTest.java

```diff
@@ -145,6 +143,23 @@ public void testDialogKeyboardTextBoxAppearanceOnStartup() {
 		checkTextBoxDimensionsAndDefaultPosition();
 	}
 
+	@Test
+	public void testConfirmButton() {
+		enterTestText();
+		selectFormatting(FormattingOptions.SERIF);
```

Comment:
No need to test unrelated properties. This will make it harder to adapt functionality, if everything is interwoven.